server/src/component/SwaggerComponent.hh

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <oatpp/core/macro/component.hpp>
+#include <oatpp-swagger/Model.hpp>
+#include <oatpp-swagger/Resources.hpp>
+
+namespace server::component {
+
+    class SwaggerComponent {
+
+    public:
+        OATPP_CREATE_COMPONENT(std::shared_ptr<oatpp::swagger::DocumentInfo>, swagger_document_info)([] {
+            return
+                    oatpp::swagger::DocumentInfo::Builder{}
+                            .setTitle("Parking space sharing")
+                            .setDescription("Project created by trainees of Bakcyl Programowania in 2022/2023.")
+                            .setVersion("0.1.0")
+                            .addServer("http://localhost:8000", "server on localhost")
+                            .addServer("http://0.0.0.0:8000", "server on local ipv4")
```

Comment:
Also 0.0.0.0 listens on all interfaces so it isn't 'local'.

Reply:
Okay, but we should change it to "server on all IPv4 interfaces", or something like that, because it's not just "local ipv4". Same with IPv6 equivalent.

Reply:
> [IANA](https://en.wikipedia.org/wiki/Internet_Assigned_Numbers_Authority), who allocate IP addresses globally, have allocated the single IP address 0.0.0.0[[1]](https://en.wikipedia.org/wiki/0.0.0.0#cite_note-1) to [RFC](https://en.wikipedia.org/wiki/RFC_(identifier)) [1122](https://datatracker.ietf.org/doc/html/rfc1122) section 3.2.1.3.
> 
> It is named there as "This host on this network"

Quoted from [Wikipedia](https://en.wikipedia.org/wiki/0.0.0.0#Official_Standard_Meaning_and_Use)

Reply:
I guess I'll change it to "This host on this network"

Reply:
I've read up on this on `ip(7)` man page, and `0.0.0.0` is said to "bind to all *local* interfaces" so maybe we should leave it as it is with "server on local ipv4". Sorry for misleading.